### ISSUE TITLE
'ubuntu-16.04' runner will be deprecated, upgrade suggested

### ISSUE THREAD

**Issue by evianzhow** [helaili/jekyll-action#107]
Source: https://github.blog/changelog/2021-04-29-github-actions-ubuntu-16-04-lts-virtual-environment-will-be-removed-on-september-20-2021/

I suddenly got following messages when executing my own repo jobs:

```
Can't find any online and idle self-hosted runner in the current repository, account/organization that matches the required labels: 'ubuntu-16.04'
Waiting for a self-hosted runner to pickup this job...
```

Change to `ubuntu-latest` works fine. 

So, I suggested moving to `ubuntu-latest` if we are not pinned to 16.04 LTS for specific reason?

**Comment by helaili** [comment]
Where do you see `ubuntu-16.04` being used?

**Comment by evianzhow** [comment]
@helaili  I didn't search the repo for `ubuntu-16.04`, sorry for that. Seem like you have already using `ubuntu-latest` instead. 

But indeed I found this example misleading, which cause me this issue: https://github.com/helaili/jekyll-action/blob/master/README.md#use-the-action

**Comment by helaili** [comment]
closed by #115
